examples/chardev.c

```diff
@@ -62,7 +63,11 @@ static int __init chardev_init(void)
 
     pr_info("I was assigned major number %d.\n", major);
 
+    #if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 4, 0)
```

Comment:
The [commit 1aaba11da9aa ("driver core: class: remove module * from class_create()
")](https://github.com/torvalds/linux/commit/1aaba11da9aa7d7d6b52a74d45b31cac118295a1) is actually v6.5 (v6.4-rc1).

Reply:
I also saw #172. Should I update to v6.5 in the patch, or this could lead to lots of #ifdefs trying to support moving code?

Reply:
I think now we still need to support v5.* kernel.
So, it's fine to add the #ifdef.

Reply:
Question on linux rc release system. v6.4-rc1 should lead to v6.4 release. 

v6.4-rc1 was released before v6.4

2023-08-27 14:49:51 -0700  (tag: v6.5)
2023-07-09 13:53:13 -0700  (tag: v6.5-rc1)
2023-06-25 16:29:58 -0700  (tag: v6.4)
2023-05-07 13:34:35 -0700  (tag: v6.4-rc1)


`$ git tag --contain 1aaba11
v6.4
v6.4-rc1
v6.4-rc2
v6.4-rc3
v6.4-rc4
v6.4-rc5
v6.4-rc6
v6.4-rc7
v6.5
v6.5-rc1
v6.5-rc2
v6.5-rc3
v6.5-rc4
v6.5-rc5
v6.5-rc6
v6.5-rc7`

Reply:
Ah, my bad.
I looked through the commit and remembered the wrong RC release system.
This PR looks good to me.